commit 6479f265cda008a6828fe68f083fbb784ef3d9dc (refs/changes/62/2762/3)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2017-10-10T22:22:45+00:00 (2 years ago)
    
    NAPI-436 Update docs for NAPI-433 (Doc typo)
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/docs/index.md b/docs/index.md
index b7d7974..c2d9318 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -687,7 +687,7 @@ List VLANs owned by a user.
 
 Create a new fabric VLAN.
 
-## Inputs
+### Inputs
 
 | Field              | Type             | Description                                                                            |
 | ------------------ | ---------------- | -------------------------------------------------------------------------------------- |
